db/migrations/202406071726_vacuum.go

```diff
@@ -15,8 +16,11 @@ import (
 var _202406071726_vacuum = &gormigrate.Migration{
 	ID: "202406071726_vacuum",
 	Migrate: func(tx *gorm.DB) error {
-		if err := tx.Exec("VACUUM").Error; err != nil {
-			return err
+		// Cannot run when testing with txdb: VACUUM must be run outside of transaction.
+		if !testing.Testing() {
```

Comment:
I think we probably don't need this migration any more. Could we just comment out the content?

Reply:
maybe commenting out the code inside the migration is fine for now